utils/gitlab_api.py

```diff
@@ -188,6 +188,46 @@ def create_app_interface_reporter_mr(self, reports, email_schema,
         return self.create_mr(branch_name, target_branch,
                               commit_message, labels=labels)
 
+    def create_app_interface_notificator_mr(
+        self, notification,
+        notification_path="/app-interface",
+        email_schema="/app-interface/app-interface-email-1.yml"
+    ):
+        # labels = ['automerge']
+        labels = ['do-not-merge/hold']
+        prefix = 'app-interface-notificator'
+        target_branch = 'master'
+
+        now = datetime.now()
+        ts = now.strftime("%Y%m%d%H%M%S")
+        isodate = now.isoformat()
+        short_date = now.strftime('%Y-%m-%d')
+
+        branch_name = 'app-interface-notificator-{}'.format(ts)
+        commit_message = '[{}] notification for {}'.format(
+            prefix, isodate
+        )
+
+        self.create_branch(branch_name, target_branch)
+
+        msg = 'add email notification'
+        email_path = f"data{notification_path}/emails/{branch_name}.yml"
```

Comment:
Using the `branch_name` to name the email file is a sign that the variable holds more than the branch name :)
Maybe rename `branch_name` to `email_id`?

By the way, you're using f-strings here! You should definetly pick one between f-strings and `format()` so it's all consistent.

Additionally, I'm always afraid of manually manipulating paths, spceially with slashs coming from different places. You forget the slash in the `notification_path` and the thing is broken. My suggestion here is to use either `os.path.join()` or `pathlib.Path()`.

Reply:
Yeah! I renamed branch_name to email_id. 

Good catch! I'll stick to f-strings then.

I see. Changed to os.path.join(). Thanks for pointing this out! ;D